### PR TITLE
Setup ReadTheDocs

### LINKED ISSUE #16: Add ReadTheDocs for the docs repo
Currently, all documentation can be browsed inside the repo.
However, it would be nice to see it in rendered form on a website.
The goal is hence to set up ReadTheDocs so that our documentation will be built automatically without much human intervention.

Story points: 2

### PR DESCRIPTION
**Pull request by delvh**
Additionally refactored all files so that there are no more warnings when generating the HTML files.
Some minor stylistic missteps have been fixed as well (CRLF line endings, file and directory names).
Result of this PR can be seen [here](https://gamifyit-docs--17.org.readthedocs.build/en/17/).

Closes #16.

### CODE REVIEW

**Review by robojones — APPROVED** [review]

**Review by Gr33ndev — APPROVED** [review]

**Review by madebyTimo — COMMENTED** [review]
nice work

File: dev-manuals/asset-packs/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # asset-packs
```
Comment (delvh) [inline dev-manuals/asset-packs/README.md:1]:
```suggestion
# Asset Packs
```

File: dev-manuals/asset-packs/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 here you can find information about where which asset pack is used
```
[the remark this answers is not in the document]
Comment (delvh) [reply dev-manuals/asset-packs/README.md:3]: Changed the title instead.

File: dev-manuals/asset-packs/README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# asset-packs
+# Asset Packs
 
 here you can find information about where which asset pack is used
-every markdown asset-pack file should follow the [template](template.md) structure.
+every markdown asset-pack file should follow the [template](asset-template.md) structure.
```
Comment (delvh) [inline dev-manuals/asset-packs/README.md:4]:
```suggestion
every markdown asset pack file should follow the [template](asset-template.md) structure.
```

**Review by madebyTimo — APPROVED** [review]

**Pull request merged**